overlay.yml

```diff
@@ -0,0 +1,24 @@
+aliases: 
+  - name: github
+    url: https://github.com/
+
+  - name: fedorapkgs
+    url: https://src.fedoraproject.org/git/rpms/
+    cacertpath: DigiCertHighAssuranceEVRootCA.crt
+
+  - name: dmi
+    url: http://copr-dist-git.fedorainfracloud.org/git/dustymabe/ignition/
+
+distgit:
+  prefix: fedorapkgs
+  branch: master
+  
+root:
+  mock: fedora-28-$arch
+
+components:
+  - src: github:projectatomic/docker
+    branch: docker-1.13.1-rhel
```

Comment:
@ashcrow, @peterbaouoft and myself tested this overlay file and it looks like we need to override the version here with `override-version: "1.13.1"`

Reply:
Updated